src/docs/asciidoc/en/disruptor.adoc

```diff
@@ -397,7 +397,7 @@ producerBarrier.commit(entry);
 ----
 
 == Throughput Performance Testing
-As a reference we choose Doug Lea’s excellent ``java.util.concurrent.ArrayBlockingQueue``footnote:arrayblockingqueue[ArrayBlockingQueue - http://download.oracle.com/javase/1.5.0/docs/api/java/util/concurrent/ArrayBlockingQueue.html] which has the highest performance of any bounded queue based on our testing.
+As a reference we choose Doug Lea’s excellent ``java.util.concurrent.ArrayBlockingQueue``footnote:arrayblockingqueue[ArrayBlockingQueue - https://docs.oracle.com/javase/8/docs/api/java/util/concurrent/ArrayBlockingQueue.html] which has the highest performance of any bounded queue based on our testing.
```

Comment:
Changed from JDK 1.5 -> 8 docs, but we're Disruptor 4.0 will be changing JDK version from 7 -> 11, so maybe pick the JDK 11 docs?